ui/app/AppLayouts/ActivityCenter/controls/NotificationCard.qml

```diff
@@ -0,0 +1,304 @@
+//     timestampText: "Yesterday"
+//     unread: false
+// }
+//
+// NOTES
+// -----
+// * Background and unread dot are theme-driven and update dynamically.
+// * Avatar, context row, and content block are self-contained components.
+// * Full-card MouseArea ensures click/hover highlight regardless of sub-content.
+// -----------------------------------------------------------------------------
+
+import QtQuick
+import QtQuick.Layouts
+import QtQuick.Effects
+import QtQuick.Controls
+
+import StatusQ.Core.Theme
+import StatusQ.Components
+import StatusQ.Core
+
+import utils
+
+Control {
+    id: root
+
+    // ──────────────────────────────────────────────────────────────────────────
+    // API
+    // ──────────────────────────────────────────────────────────────────────────
+
+    // ──────────────────────────────────────────────────────────────────────────
+    // Avatar parameters
+    // ──────────────────────────────────────────────────────────────────────────
+
+    // Avatar image URL. Empty → nothing.
+    property url avatarSource: ""
+
+    // Optional avatar badge icon URL/name. Empty → badge hidden.
+    property url badgeIconName: ""
+
+    // Render avatar as a circle when true; otherwise keep original image shape.
+    property bool isCircularAvatar: true
+
+    // ──────────────────────────────────────────────────────────────────────────
+     // Header parameters
+     // ──────────────────────────────────────────────────────────────────────────
+
+     // Title (usually display name). Truncated in the header if too long.
+     property string title: ""
+
+     // Secondary identifier (e.g., chat key: "prefix…suffix"). Shown after title.
+     property string chatKey: ""
+
+     // Shows "is contact" badge when true.
+     property bool isContact: false
+
+     // Trust level indicator (0 = none). Values from StatusContactVerificationIcons.TrustedType.
+     property int trustedIndicator: 0
+
+    // ──────────────────────────────────────────────────────────────────────────
+    // Context row parameters
+    // ──────────────────────────────────────────────────────────────────────────
+
+    // Primary context label (e.g., Community name).
+    property string primaryText
+
+    // Secondary context label (e.g., #channel).
+    property string secondaryText
+
+    // Leading/separator icon before labels. Hidden when empty.
+    property string separatorIconName
+
+    // Icon between primary and secondary labels. Hidden when empty.
+    property string iconName
+
+    // ──────────────────────────────────────────────────────────────────────────
+    // Action and meta data parameters
+    // ──────────────────────────────────────────────────────────────────────────
+
+    // Short hint or action text displayed below the context row.
+    property string actionText: ""
+
+    // Timestamp text (e.g., "2h", "Yesterday"). Defaults to "Just now" if empty.
+    property string timestampText: ""
```

Comment:
Not sure here... I'd rather take the `timestamp` as a `double` and do the conversion/formatting inside this component, it's truly an implementation detail. But can be done later :)